ParisBNF/abb/BNFabb203.xml

```diff
@@ -299,26 +292,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
                         <bindingDesc>
                            <binding contemporary="unknown" cert="low">
-                              <decoNote xml:id="b1">Binding probably Ethiopian,
-                                but not fully typical.</decoNote>
-                               <decoNote xml:id="b2">Two wooden boards.</decoNote>
-                                   <decoNote xml:id="b3">Blind-tooled leather cover, with the
+                              
+                             <decoNote xml:id="b1" type="Boards">Binding probably Ethiopian
```

Comment:
b1 has no type, no type is printed for b1, see guidelines

Reply:
Indeed, thanks.